src/Components/Calculator.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
 import React, { useState } from "react";
 import "../Styles/Calculator.css";
```

Comment:
- [ ] Good job so far 👍 You are required to use class-based components here. Kindly refactor your `Calculator.js` component into a class-based component and you'll be good to go 💯